addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -714,6 +716,9 @@ export function useOpenX2ManyRecord({
             userService,
             env,
         });
+        if (component.props.record.config.mode === "readonly") {
```

Comment:
use isInEdition instead? config is kinda private (even though we didn't prefix it with _)